File: CS/WindowsApplication41/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Diagnostics;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace WindowsApplication41
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }
        private void Form1_Load(object sender, EventArgs e)
        {
            for (int i = 0; i < 6; i++)
                dataTable1.Rows.Add(new object[] { i });
        }

        private void simpleButton1_Click(object sender, EventArgs e) {
            ratingControl1.Position += 1;
        }
    }
}

File: VB/WindowsApplication41/Form1.vb
Imports System
Imports System.ComponentModel
Imports System.Diagnostics
Imports System.Drawing
Imports System.Windows.Forms

Namespace WindowsApplication41

    Public Partial Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs)
            For i As Integer = 0 To 6 - 1
                dataTable1.Rows.Add(New Object() {i})
            Next
        End Sub

        Private Sub simpleButton1_Click(ByVal sender As Object, ByVal e As EventArgs)
            ratingControl1.Position += 1
        End Sub
    End Class
End Namespace
